js/main.js

```diff
@@ -0,0 +1,281 @@
+const $ul = document.querySelector('ul');
+const $modal = document.querySelector('#modal-section');
+const $class = document.querySelector('#classes-box');
+const $classBox = document.querySelectorAll('.class-box');
+
+$ul.addEventListener('click', handleOperator);
+$class.addEventListener('click', handleFilter);
+
+const xhr = new XMLHttpRequest();
+xhr.open(
+  'GET',
+  'https://lfz-cors.herokuapp.com/?url=https://api.rhodesapi.com/api/operator?exclude=talents,rarity,artist,va,description,quote,voicelines,alter,affiliation,tags,range,statistics,trait,potential,trust,skills,costs,module,base,headhunting,recruitable,availability,release_dates',
+);
+xhr.responseType = 'json';
+xhr.addEventListener('load', function () {
+  console.log(xhr.status);
+  console.log(xhr.response);
```

Comment:
Remove all `console.log`s before submitting. These are only for testing / debugging purposes